index.html

```diff
@@ -38,14 +36,63 @@
   <link rel="preconnect" href="https://fonts.googleapis.com" />
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;700&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="css/style.css" type="text/css" media="screen" />
+  <link rel="stylesheet" href="css/faq.css">
 </head>
 
 <body>
   <!-- This div pushes down header a little bit so that header is not sticky from start -->
   <div class="header-push-down"></div>
 
-  <header class="blur disable-selection">
-    <nav>
+  <header>
+
+    <nav class="navbar navbar-expand-lg" data-bs-theme="dark">
+      <div class="container-fluid">
+  
+        <div class="left navbar-title">
+          <img src="images/icon/icon.png" alt="OpenCodeEra logo" />
+          <a class="navbar-brand" href="index.html">
+            <span class="blue">Open</span>
+            <span class="white">Code</span>
+            <span class="blue">Era</span>
+          </a>
+          <!-- <a  href="#">Open Code Era</a> -->
+        </div>
+      
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav me-0 mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link" aria-current="page" href="#">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link about" href="#about">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link services" target="_blank" href="https://github.com/orgs/OpenCodeEra/repositories">Projects</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="team_list.html">Team</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link contact-us" href="#contact">Contact Us</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link about" href="#faq">FAQ</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  
+  </header>
```

Comment:
Hey @arujjval, the navbar is responsive, but if you style the navbar menu, then it will be great.

Right now it looks like this :-

![image](https://github.com/OpenCodeEra/OpenCodeEra/assets/83089446/6e8b01ab-a92c-4e20-b567-53fe7db8af95)